pyanaconda/modules/payloads/payload/dnf/dnf.py

```diff
@@ -245,6 +256,25 @@ def _set_up_kickstart_packages_selection(self, data):
             )
             data.packages.excludedGroupList.append(group)
 
+        for name in selection.modules:
+            self._set_up_kickstart_module_data(data, name)
+
+        for name in selection.disabled_modules:
+            self._set_up_kickstart_module_data(data, name, False)
+
+    @staticmethod
+    def _set_up_kickstart_module_data(data, name, enabled=True):
+        """Set up the kickstart data for the module command."""
+        names = name.split(":", maxsplit=1)
+        names += ["", ""]
```

Comment:
Maybe adding just [""] would be enough here IIUIC, or are two "" items safer or more readable?
Hm, sounds like friday nitpicking, feel free to ignore...

Reply:
Fixed.